skrifa/src/outline/cff/mod.rs

```diff
@@ -87,6 +91,15 @@ impl<'a> Outlines<'a> {
         matches!(self.version, Version::Version2(_))
     }
 
+    /// Returns the number of available glyphs.
+    pub fn len(&self) -> usize {
```

Comment:
Optional: I don't find it obvious that this is what len of Outlines would mean, perhaps a more specific name, like subfont_count did, would make sense? glyph_count for example is painfully clear.

Reply:
Changed to glyph_count (and did the same for glyf::Outlines)